### ISSUE TITLE
Contact Form php still sends blank emails

### ISSUE THREAD

**Issue by brianne-king** [brianne-king/mywebsite#1]
I am having a simple issue with my bootstrap contact form. It sends me a blank email every day (spambot I'm guessing) and I want it to stop. I've considered using a CAPTCHA but I honestly just want the blank email to stop coming in. I'm using HostGator for hosting and they provide me with good spambot security but still can't seem to get this issue resolved. PLEASE HELP!

This is my HTML code:
 <section id="contact" class="contact bg-primary">
        <div class="container">
            <h2 class="text-center">Contact Me</h2>
            <hr class="star-primary">
            <div class="row">
                <div class="col-lg-12 offset-lg-2">
                    <form  method="post" action="mail/contact_me.php" name="sentMessage" id="contactForm" novalidate>
                           <div class="control-group">
                            <div class="form-group floating-label-form-group controls">
                               
                                <input class="form-control" name="name" id="name" type="text" placeholder="Name" required data-validation-required-message="Please enter your name.">
                                <p class="help-block text-danger"></p>
                            </div>
                        </div>
                        <div class="control-group">
                            <div class="form-group floating-label-form-group controls">
                                <input class="form-control" name="email" id="email" type="email" placeholder="Email Address" required data-validation-required-message="Please enter your email address.">
                                <p class="help-block text-danger"></p>
                            </div>
                        </div>
                        <div class="control-group">
                            <div class="form-group floating-label-form-group controls">
                                <input class="form-control" name="phone" id="phone" type="tel" placeholder="Phone Number" required data-validation-required-message="Please enter your phone number.">
                                <p class="help-block text-danger"></p>
                            </div>
                        </div>
                        <div class="control-group">
                            <div class="form-group floating-label-form-group controls">
                                <textarea name="message" id="message" class="form-control" rows="5" cols="5" placeholder="Message" required data-validation-required-message="Please enter a message."></textarea>
                                <p class="help-block text-danger"></p>
                            </div>
                        </div>
                        <br>
                        <div id="success">
                        <div class="form-group">
                            <button type="submit" class="btn btn-success btn-lg">Send</button>
                            </div>
                        </div>
                    </form>
            </div>
        </div>
		</div>
    </section>

Here is my PHP:

![d373d01d1c76f4df3807661512c17e68](https://user-images.githubusercontent.com/33229852/32183811-4ca6ff1c-bd68-11e7-9493-71d2c8b91211.png)

**Comment by brianne-king** [comment]
**Also if you have suggestions for something alot simpler I am all ears.**

**Comment by DynamiteC** [comment]
Let me try, it would be my first try so might take a while.

**Comment by DynamiteC** [comment]
Their is error, "The requested URL /mywebsite-master/mail/contact_me.php was not found on this server." so to solve this issues i would required your all file.

**Comment by brianne-king** [comment]
Yes I am sorry I do not believe I set this up correctly. It is my first time using Github.

**Comment by DynamiteC** [comment]
No worries it happens to the best. Upload all the files.
